battenberg/core.py

```diff
@@ -186,21 +190,24 @@ def install(self, template, checkout='master', extra_context=None, no_input=Fals
         # Let's merge our changes into HEAD
         self.merge_template_branch(f'Installed template \'{template}\'')
 
-    def upgrade(self, checkout='master', extra_context=None, no_input=False):
+    def upgrade(self, checkout: str = 'master', extra_context: Dict = None, no_input: bool = False,
```

Comment:
Should we require user confirmation before proceeding with an upgrade? The default is usually still master, right?

Reply:
Added in https://zbrt.atl.zillow.net/browse/ZOMLINFRA-425 to cover this. Will add this if I get time in the sprint.